src/js/views/patients/schedule/schedule_views.js

```diff
@@ -160,14 +160,19 @@ const DayItemView = View.extend({
     });
   },
   onRender() {
+    this.toggleSelected(!this.isReduced && this.state.isSelected(this.model));
```

Comment:
This ensures `.is-selected` is properly added on the initially route load or when the user uses the bulk select controls to select all or un-select all.

The `!this.isReduced` check is added here because the state model is not loaded for the `reduced_schedule_app.js`, which means `this.state.isSelected()` doesn't exist and throws an error when the reduced app is loaded.